bee-trace/bee-trace-attributes/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "bee-trace-attributes"
+version = "0.1.0"
+edition = "2021"
+description = "Procedural macros for bee-trace"
+readme = "README.md"
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = [ "iota", "tangle", "bee", "framework", "trace", "tracing", "attribute", "proc-macro" ]
+homepage = "https://www.iota.org"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+proc-macro2 = { version = "1.0.28", default-features = false }
+quote = { version = "1.0.9", default-features = false }
+syn = { version = "1.0.75", default-features = false, features = [ "full", "extra-traits", "parsing", "printing", "derive", "proc-macro", "clone-impls" ] }
+
+[dev-dependencies]
+bee-trace = { path = "../bee-trace" }
```

Comment:
default-feature = false on dev deps as well, if it can help reduce CI time, why not...